ajax/libs/colresizable/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "colresizable",
+  "filename": "colResizable-1.6.min.js",
+  "version": "1.6.0",
+  "description": "jQuery plugin to resize table columns",
+  "author": "Alvaro Prieto Lauroba",
+  "homepage": "http://bacubacu.com/colresizable/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/alvaro-prieto/colResizable.git"
+  },
+  "keywords": [
+    "resize table",
+    "resize columns",
+    "resize",
+    "resizable",
+    "columns",
+    "jquery-plugin",
+    "table",
+    "range slider",
```

Comment:
Let's update this to only `slider` and remove other keywords with spaces
The reason is, for example, 
when you type `resize table`, it matched `resize` and `table`, so there is no need to use space in the `keywords`.

Reply:
@extend1994 So what should I do with autoupdate part of package.json? Should I remove it temporary until we will get answer from author about next releases?